src/App.js

```diff
@@ -1,27 +1,59 @@
-import logo from "./logo.svg";
 import "./App.css";
 import { SkynetProvider } from "./state/SkynetContext";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
```

Comment:
You'll want to change "BrowserRouter" to "HashRouter" for Skynet Deployments. (At least until our support for this rolls out in the next couple of weeks.)